### DISCUSSION TITLE
When setting Pageview goal, can't use **keyword** like docs show

### DISCUSSION THREAD

**Discussion by steevka** [plausible/analytics#1925] [Anything Else]
Hey all, 

New to Plausible but loving it so far. I was reading through the Goals docs and came across this: 

https://plausible.io/docs/pageview-goals

> **keyword** | use double  asterisks in front and back in the same way that you would use  "contain". It finds any URL containing a specific keyword

I tried to set a new Pageview goal trigger using the following: `**successful**`

But I get the error: 

> this field is required and must start with a /

See picture below.

Am I misunderstanding the wildcard keyword function?

Thanks!

**Comment by steevka** [comment]
![Screen Shot 2022-05-25 at 5 24 16 PM](https://user-images.githubusercontent.com/11139132/170370431-acc5527f-fc63-4753-9866-8f99a127739d.png)

**Comment by steevka** [comment]
Nevermind, I just had to do `/**successful**`

**Reply by metmarkosaric** [reply]
Thanks for the update @steevka! Enjoy Plausible!
